websites/R/Rythm/presence.ts

```diff
@@ -483,3 +405,215 @@ function getPageletiable(js: string): Promise<string> {
+        });
+      }
+
+      // Check if the song isn't paused
+      if (apiData.pause == false) {
+        // Add the Rythm version name and seperator
+        presenceData.smallImageText = `${rythms[namespace]} ${separator} `;
+
+        // Check repeating state and add it
+        switch (apiData.repeat) {
+          // Repeat song
+          case "one":
+            presenceData.smallImageKey = "repeat-one";
+            presenceData.smallImageText += (await strings).repeat;
+            break;
+          // Repeat queue
+          case "queue":
+            presenceData.smallImageKey = "repeat";
+            presenceData.smallImageText += (await strings).repeatAll;
+            break;
+          // Play
+          default:
+            presenceData.smallImageKey = "play";
+            presenceData.smallImageText += (await strings).play;
+            break;
+        }
+
+        // Check if the currently playing song is a livestream
+        if (apiData.isStream) {
+          // Show elapsed time
+          presenceData.startTimestamp = apiData.seekTimestamp / 1000;
+        } else {
+          // Show remaining time
+          presenceData.startTimestamp = (apiData.seekTimestamp - apiData.seek) / 1000;
+          presenceData.endTimestamp = (apiData.seekTimestamp - apiData.seek + apiData.length) / 1000;
+        }
+      } else {
+        // Pause
+        presenceData.smallImageKey = "pause";
+        presenceData.smallImageText = rythms[namespace] + ` ${separator} ` + (await strings).pause;
+      }
+    } else {
+      // Check if server details can and should be shown
+      if (showServerDetails && apiData.guildDisplayName) {
+        // Get the amount of DJs and normal users
+        const djAmount = apiData.users.filter(user => user.isAdmin).length,
+          userAmount = apiData.users.length - djAmount;
+
+        // Add server information
+        presenceData.details = apiData.guildDisplayName;
+
+        // Check if any users are connected
+        if (djAmount > 0 || userAmount > 0) {
+          // Add user information
+          presenceData.state =
+            `${(await strings).dj}s: ${djAmount}` +
+            ` ${separator} ` +
+            `Users: ${userAmount}`;
+        }
+      } else {
+        // Check the idle time
+        if (idleStamp == 0) idleStamp = Date.now() / 1000;
+
+        // Add idle information
+        presenceData.details = "Idle";
+        presenceData.startTimestamp = idleStamp;
+      }
+    }
+
+    presence.setActivity(presenceData);
+    presence.setTrayTitle(`${namespace} ${separator} Dashboard`);
+    return;
+  }
+
+  // Documentation
+  if (path[0] == "docs") {
+    presenceData.largeImageKey = "r1";
+    presenceData.details = "Documentation";
+    presenceData.state = document.querySelector("h1").textContent;
+
+    presence.setActivity(presenceData);
+    presence.setTrayTitle(`Rythm ${separator} Documentation`);
+    return;
+  }
+
+  // Premium
+  if (path[0] == "premium") {
```

Comment:
maybe you could use `switch(path[0])` instead of `if (path[0] == ...)` 🤔

Reply:
This would be a really big switch statement and might make it confusing, that's why I decided to go with the normal if statement